app/assets/stylesheets/was_seed.scss

```diff
@@ -14,9 +13,12 @@
 .#{$namespace}-was-thumb-item {
   cursor: pointer;
   float: left;
-  height: 100%;
   margin: 10px;
-  width: 200px;
+}
+
+.#{$namespace}-was-thumb-item-img {
+    display:block;
+    margin:auto;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Colon after property should be followed by one space